<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseForm</class>
 <widget class="QWidget" name="DatabaseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_langLabel">
       <property name="text">
        <string>Athkar language:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_langList"/>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="m_showMeritsBox">
       <property name="text">
        <string>Show merits</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
